ocp_resources/image_stream.py

```diff
@@ -43,18 +43,21 @@ def to_dict(self):
         res = super().to_dict()
         if self.yaml_file:
             return res
-        res.update(
-            {
-                "spec": {
-                    "lookupPolicy": {"local": self.lookup_policy},
-                    "tags": {
-                        "from": {"kind": "DockerImage", "name": self.image_repository},
-                        "name": self.tag,
-                        "referencePolicy": {"type": "Source"},
-                    },
-                }
+        tags = []
+        tag_name = self.tag
+        repository_name = self.image_repository
+        for x, y in zip(tag_name, repository_name):
```

Comment:
I think the user should pass the tags dict.

Reply:
Fixed.